docs/dev/python/table-api-users-guide/udfs/python_udfs.md

```diff
@@ -25,7 +25,7 @@ under the License.
 
 User-defined functions are important features, because they significantly extend the expressiveness of Python Table API programs.
 
-**NOTE:** Python UDF execution requires Python version (3.5, 3.6 or 3.7) with PyFlink installed. It's required on both the client side and the cluster side. 
+**NOTE:** Python UDF execution requires Python version (3.5, 3.6, 3.7 or 3.8) with PyFlink installed. It's required on both the client side and the cluster side. 
```

Comment:
The corresponding content of `python_udfs.zh.md` also needs to be modified together